apps/api/src/.env.production

```diff
@@ -14,6 +14,9 @@ REDIS_PORT=6379
 REDIS_PREFIX=
 REDIS_DB_INDEX=2
 
+REDIS_CACHE_HOST="localhost"
+REDIS_CACHE_PORT="6379"
```

Comment:
Will the cache potentially be in a different instance of the current Redis database?

Reply:
@p-fernandez It can be. And in my opinion, we should separate because most of our load is when we have a trigger calculation and due to that reason uses the current instance in order to maintain the jobs data while most of our requests to the dal are when processing the trigger event it will be a good idea to separate them IMO.